docs/adrs/0019-ui-tests-against-dev.md

```diff
@@ -0,0 +1,44 @@
+# 19. Use Dev Environment for UI Tests
+
+**Date**: 2024-10-14  
+
+## Status
+
+Accepted
+
+## Context
+
+Previously, we had two types of Cypress tests for our UI:
+
+- **Deployment Tests**: These pointed to the live URL of our Dev environment.
```

Comment:
Also to the Test environment. They were lightweight and the sole purpose was to ensure that a deployment hadn't broken the whole system